include/reloader.h

```diff
@@ -89,6 +82,15 @@ class Reloader {
 	/// notification will contain \a msg passed.
 	void notify(const std::string& msg);
 
+protected:
```

Comment:
Class members are private by default, so this commit effectively relaxed the access. No classes inherit from `Reloader`, so the point is kind of moot, but still: why change the access level?

Reply:
My bad, will fix it and force-push this small change.